tree/tree/inc/ROOT/RFriendInfo.hxx

```diff
@@ -77,18 +77,18 @@ struct RFriendInfo {
    RFriendInfo(std::vector<std::pair<std::string, std::string>> friendNames,
                std::vector<std::vector<std::string>> friendFileNames,
                std::vector<std::vector<std::string>> friendChainSubNames,
-               std::vector<std::vector<std::int64_t>> nEntriesPerTreePerFriend,
+               std::vector<std::vector<Long64_t>> nEntriesPerTreePerFriend,
                std::vector<std::unique_ptr<TVirtualIndex>> treeIndexInfos);
 
    void AddFriend(const std::string &treeName, const std::string &fileNameGlob, const std::string &alias = "",
-                  std::int64_t nEntries = std::numeric_limits<std::int64_t>::max(), TVirtualIndex *indexInfo = nullptr);
+                  Long64_t nEntries = std::numeric_limits<Long64_t>::max(), TVirtualIndex *indexInfo = nullptr);
```

Comment:
Here and in similar places, maybe we could have 
```suggestion
                  Long64_t nEntries = TTree::kMaxEntries, TVirtualIndex *indexInfo = nullptr);
```
instead?

Reply:
would need to check the code for places where we match this value, e.g. in TTreeProcessorMT there is a `fGlobalRange.second != std::numeric_limits<Long64_t>::max()`. and I don't really understand https://github.com/root-project/root/pull/12352 so I'm not sure we'd incur in it here (@hahnjo ?).

Reply:
`static constexpr Long64_t kMaxEntries = std::numeric_limits<Long64_t>::max();`

So it really is a semantic choice.

Reply:
that doesn't really change the fact that we want to be consistent in which one we use

Reply:
If your number is a number of entries, I recommend using `TTree::kMaxEntries`